src/browser/vdiff.js

```diff
@@ -1,11 +1,20 @@
 import { chai, expect } from '@open-wc/testing';
 import { executeServerCommand } from '@web/test-runner-commands';
 
-chai.Assertion.addMethod('golden', ScreenshotAndCompare);
+mocha.setup({ // eslint-disable-line no-undef
+	rootHooks: {
+		beforeEach() {
+			const { currentTest } = this;
+			chai.Assertion.addMethod('golden', function(...args) {
```

Comment:
Any worry about adding the method over and over again? I guess Chai will just overwrite it. Would something like this also work, or does that lose the "this"?

```javascript
let currentTest, obj;

chai.Assertion.addMethod('golden', (...args) => {
	return ScreenshotAndCompare.call({ test: currentTest, elem: obj }, ...args); 
});
mocha.setup({ // eslint-disable-line no-undef
	rootHooks: {
		beforeEach() {
			currentTest = this.currentTest;
			obj = this._obj;
		}
	}
});
```

Reply:
It seems to just overwrite it, but I agree it feels bad.

Different `this`-es, but this would probably work:
```javascript
let test;

chai.Assertion.addMethod('golden', function(...args) {
	return ScreenshotAndCompare.call({ test, elem: this._obj }, ...args); 
});
mocha.setup({ // eslint-disable-line no-undef
	rootHooks: {
		beforeEach() {
			test = this.currentTest;
		}
	}
});
```